internal/utility/utility.go

```diff
@@ -0,0 +1,20 @@
+package utility
```

Comment:
@chronark it'd be simpler to use a 3rd-party library for this, but it didn't seem worth adding an extra dependency for something so small. Feel free to swap this out / move it if you'd like to structure things differently.

Reply:
Yeah I agree, no reason to bring in a dependency for this.
I'm thinking about moving it to `/pkg/util` though, what do you think?

Great work, thank you.